src/Infrastructure/TemplateMailerInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Infrastructure;
+
+use WMDE\Fundraising\Frontend\MembershipContext\Domain\Model\EmailAddress;
+
+interface TemplateMailerInterface {
```

Comment:
@KaiNissen offered a formidable explanation, thank you. @JeroenDeDauw Which solution do you propose?